### ISSUE TITLE
Flag to build updated crates in debug mode

### ISSUE THREAD

**Issue by dtolnay** [nabijaczleweli/cargo-update#49]
The `cargo install` command has a `--debug` flag, as in `cargo install mdbook --debug`. So something like:

```sh
cargo install-update mdbook --debug
```

This is useful for tools that compile slowly in release mode but run fast enough even in debug mode.

**Comment by nabijaczleweli** [comment]
As of 3823f08566fb9467d7466bdcdd0e4b93488cb26c you can use `install-update-config`:

```shell
cargo install-update-config mdbook --debug
```

Which will set `mdbook`  to compile in debug mode. Reflexively, use `--release` to disable.

**Comment by nabijaczleweli** [comment]
Released in [`v1.2.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v1.2.0).
